src/entrypoints/options/pages/translation/translation-config.tsx

```diff
@@ -0,0 +1,212 @@
+import type { PageTranslateRange, TranslateProviderNames } from '@/types/config/provider'
+import deepmerge from 'deepmerge'
+
+import { useAtom, useAtomValue } from 'jotai'
+import ProviderIcon from '@/components/provider-icon'
+import { Checkbox } from '@/components/ui/checkbox'
+import { Input } from '@/components/ui/input'
+import {
+  Select,
+  SelectContent,
+  SelectGroup,
+  SelectItem,
+  SelectLabel,
+  SelectTrigger,
+  SelectValue,
+} from '@/components/ui/select'
+import { isLLMTranslateProvider, pageTranslateRangeSchema, translateProviderModels } from '@/types/config/provider'
+import { configFields } from '@/utils/atoms/config'
+import { LLM_TRANSLATE_PROVIDER_ITEMS, PURE_TRANSLATE_PROVIDER_ITEMS } from '@/utils/constants/config'
+import { ConfigCard } from '../../components/config-card'
+import { FieldWithLabel } from '../../components/field-with-label'
+import { SetApiKeyWarning } from '../../components/set-api-key-warning'
+
+export default function TranslationConfig() {
+  return (
+    <ConfigCard title={i18n.t('options.general.translationConfig.title')} description={i18n.t('options.general.translationConfig.description')}>
+      <div className="space-y-4">
+        <TranslateProviderSelector />
+        <TranslateModelSelector />
+        <RangeSelector />
```

Comment:
only range selector should be moved to 	`translation page`, 

`<TranslateProviderSelector />` and `<TranslateModelSelector />` should remain in the general home page (because users are very likely wanna find these two settings when they open the option page

so move <RangeSelector /> and Wrapper it with a ConfigCard with title and description.

`main content` means the major dom except `footer` `header` stuff. `All content` means, literally, all dom node (you can add this to description of the ConfigCard)

Reply:
![Uploading CleanShot 2025-05-29 at 17.08.19@2x.png…]()

Reply:
![CleanShot 2025-05-29 at 17 08 19@2x](https://github.com/user-attachments/assets/20163ac5-a0e0-427a-8ad7-d358c72acace)